src/js/control-bar/control-bar.js

```diff
@@ -67,6 +68,7 @@ ControlBar.prototype.options_ = {
     'descriptionsButton',
     'subtitlesButton',
     'captionsButton',
+    'tracksButton',
```

Comment:
What about removing from here and updating the combined-tracks test page to include all the separate caption buttons in the control bar as a default config?